packages/angular/src/date-time-picker/date-time-relative.component.ts

```diff
@@ -50,12 +51,12 @@ export const getRangeFromRelative = (relativeConfig: RelativeRange): DateRange =
     <fieldset class="bx--fieldset iot--date-time-picker__menu-formgroup">
       <legend class="bx--label">Relative to</legend>
       <div class="iot--date-time-picker__fields-wrapper">
-        <ibm-select class="bx--form-item" [(ngModel)]="relativeTo" (valueChange)="onChange()">
+        <ibm-select class="bx--form-item" [(ngModel)]="relativeTo" (valueChange)="onChange()" theme="light">
           <option value="YESTERDAY" selected>Yesterday</option>
         </ibm-select>
         <!-- tmp until we can implement a better time selector -->
         <div class="bx--form-item">
-          <input ibmText type="time" [(ngModel)]="relativeTime" (change)="onChange()" />
+          <input ibmText type="time" [(ngModel)]="relativeTime" (change)="onChange()" theme="light"/>
```

Comment:
does this break any dark theme stuff? wasn't sure

Reply:
Nope, confusingly the "light" theme here is just to give the form controls more contrast against the background (it changes with the theme)